testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergProcedureCalls.java

```diff
@@ -56,6 +57,68 @@ public void testMigrateHiveTable()
         onTrino().executeQuery("DROP TABLE IF EXISTS " + icebergTableName);
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testMigrateTimestampMillisHiveTableWithOrc()
+    {
+        String tableName = "test_migrate_timestamp" + randomNameSuffix();
+        String hiveTableName = "hive.default." + tableName;
+        String icebergTableName = "iceberg.default." + tableName;
+        String sparkTableName = "iceberg_test.default." + tableName;
+
+        onTrino().executeQuery("DROP TABLE IF EXISTS " + hiveTableName);
+        onTrino().executeQuery("CREATE TABLE " + hiveTableName + " (x timestamp(3)) WITH (format='ORC')");
+        onTrino().executeQuery("INSERT INTO " + hiveTableName + " VALUES timestamp '2021-01-01 10:11:12.123'");
+
+        onTrino().executeQuery("CALL iceberg.system.migrate('default', '" + tableName + "')");
+
+        assertThat(onTrino().executeQuery("SELECT x FROM " + icebergTableName))
+                .containsOnly(row(Timestamp.valueOf("2021-01-01 10:11:12.123")));
+        onSpark().executeQuery("SET `spark.sql.iceberg.handle-timestamp-without-timezone` = TRUE");
```

Comment:
why? document (here & below)

Reply:
worth adding a word of comment as a code comment?

Reply:
I added but I have not pushed it yet as the product tests were running on my laptop.